hhs_facilities/delphi_hhs_facilities/constants.py

```diff
@@ -1,14 +1,22 @@
-"""Registry for signal names and geographies to process."""
+"""Registry for signals and geographies to process."""
+from .generate_signals import sum_cols
 
-TOTAL_BEDS_7DAV = "total_beds_7_day_avg"
+NAN_VALUE = -999999.0
+
+ADMISSION_TOTAL = "total_admissions_7dav"
```

Comment:
total nit but since you're not dividing by 7 to get an average,
```suggestion
ADMISSION_TOTAL = "total_admissions_7d"
```

Reply:
thanks, apparently my mind has gotten stuck on thinking everything thats 7 days is an average.